alacritty_terminal/src/input.rs

```diff
@@ -692,6 +694,12 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
             self.mouse_report(code, ElementState::Released, modifiers);
             return;
         } else if let (Some(point), true) = (point, button == MouseButton::Left) {
+            let mouse_state = self.mouse_state(point);
+            self.update_mouse_cursor(mouse_state);
+            if let MouseState::Url(url) = mouse_state {
+                let url_bounds = url.linear_bounds(self.ctx.terminal());
+                self.ctx.terminal_mut().set_url_highlight(url_bounds);
+            }
```

Comment:
Why is this required here? The mouse highlight is accurate for me after clicking on URLs even without this.

Reply:
Ah, got it!

Reply:
This makes me think though, couldn't we move the `self.launch_url()` call into this then and remove the check for modifiers at the start of the `launch_url` method?